graylog2-web-interface/src/views/pages/ExtendedSearchPage.jsx

```diff
@@ -89,7 +107,7 @@ const ViewAdditionalContextProvider = connect(AdditionalContext.Provider, { view
 const ExtendedSearchPage = ({ route, searchRefreshHooks }: Props) => {
   const refreshIfNotUndeclared = view => _refreshIfNotUndeclared(searchRefreshHooks, SearchExecutionStateStore.getInitialState(), view);
 
-  useEffect(() => {
+  React.useEffect(() => {
```

Comment:
I like `useEffect` directly imported more. What is the reason for changing  it?

Reply:
@konrad I changed it, because I need to use `React.ComponentType`
I also prefer the direct import. Are you fine with:
```
import React, { useEffect } from 'react';
import type { ComponentType } from 'react';
```
Feels a bit strange to import from react multiple times, but I could deal with it.

Reply:
Thats how we do it on other places! So I am fine with it. :+1: